scss/media.scss

```diff
@@ -0,0 +1,428 @@
+      justify-content: space-between;
+      width: 100%;
+      a {
+        color: white ;
+        position: relative;
+        min-width: 85px;
+        &:hover {
+          color: $hoverColorLink;
+        }
+      }
+      .currentLink {
+        &::after {
+          content: "";
+          position: absolute;
+          margin-top: 20px;
+          left: 25%;
+          width: 40px;
+          height: 2px;
+          background-color: white;
+        }
+      }
+    }
+    .enterButton {
+      position: static ;
+      flex-direction: row ;
+      margin-bottom: 0 ;
+      margin-right: 70px;
+      button {
+        margin-top: 20px ;
+        margin-right: 20px ;
+        background-color: transparent;
+        border: 1.5px solid white;
+        color: white ;
+        &:hover {
+          color: $hoverColorLink;
+          border: 1.5px solid $hoverColorLink;
+        }
+      }
+    }
+  }
+
+  .headerText {
+    margin-top: 0;
+    position: absolute;
+    top: 37%;
+    left: 28%;
+    span {
+      font-size: 110px;
+      line-height: 130px;
+    }
+  }
+  .container {
+    position: relative;
+    min-height: 500px;
+  }
+
+  .tabs {
+    width: 770px;
+    height: 80px;
+    margin-top: -35px;
+    a {
+      font-size: 12px;
+      line-height: 24px;
+      padding-top: 4%;
+    }
+  }
+
+  .accordionsContainer {
+    margin-top: 98px !important;
+  }
+
+  #headTips {
+    width: 247px !important;
+    height: 41px !important;
+    margin-top: 19px !important;
+    margin-bottom: 95px !important;
+  }
+
+  .accordContainer {
+    min-height: 151px !important;
+    position: relative;
+    .headAccordion {
+      .number {
+        font-size: 110px !important;
+        line-height: 130px !important;
+        width: 50px !important;
+        margin-right: 113px !important;
+      }
+      .accordText {
+        font-size: 24px;
+        line-height: 36px;
+      }
+      .accordion {
+        width: 37.5px !important;
+        height: 35px !important;
+        &::after {
+          width: 20px !important;
+          height: 2px !important;
+          top: 49% !important ;
+          left: 20% !important;
+        }
+        &::before {
+          width: 20px !important;
+          height: 2px !important;
+          top: 49% !important;
+          transform: rotate(90deg);
+        }
+      }
+    }
+  }
+
+  .accordionActive {
+    position: absolute;
+    right: 0;
+    top: -10px;
+    margin-top: 57px;
+    width: 35px;
+    height: 35px;
+    margin-left: 37px;
+    &::after {
+      width: 20px !important;
+      height: 2px !important;
+    }
+  }
+
+  .accordTextActive {
+    flex-basis: 15% !important;
+    margin-top: 57px;
+    font-size: 42px !important;
+    line-height: 54px !important;
+  }
+
+  .panel {
+    position: relative;
+    .mainIcon {
+      margin: 42px 0 0 0 !important;
+    }
+    .footerAccordion {
+      font-size: 16px;
+      margin-bottom: 63px;
+    }
+    .footerAccordionContainer {
```

Comment:
this element shouldn't be positioned as absolute

Reply:
done